Project.toml

```
@@ -31,3 +31,8 @@ Profile = "1"
 Reactant = "0.2.50"
 Serialization = "1"
 Setfield = "1.1.2"
+
+[sources]
+# XXX: Remove before merging
+Reactant = {url = "https://github.com/EnzymeAD/Reactant.jl", rev = "ap/more_sharding_niceties"}
```

I can't find this branch (was probably from an old PR?).  I presume it'd be helpful to use
```suggestion
Reactant = {url = "https://github.com/simone-silvestri/Reactant.jl", rev = "main"}
```
to use https://github.com/EnzymeAD/Reactant.jl/pull/1013 instead?